uberfire-api/src/main/java/org/uberfire/backend/vfs/impl/ObservablePathImpl.java

```diff
@@ -293,7 +293,7 @@ private void executeRenameCommands() {
         }
     }
 
-    private void executeConcurrentRenameCommand(final Path path,
+    void executeConcurrentRenameCommand(final Path path,
```

Comment:
This class should be reformatted too (as removal of the access modifier means the parameters are no longer aligned.... arguably petty, but code style should be maintained to avoid spurious _white space_ diffs in the future).

Reply:
done thnx, sorry for inattention